apps/liteprotocoltester/tester_message.nim

```diff
@@ -0,0 +1,111 @@
+when (NimMajor, NimMinor) < (1, 4):
+  {.push raises: [Defect].}
+else:
+  {.push raises: [].}
+
+import
+  chronicles,
+  json_serialization,
+  json_serialization/std/options,
+  json_serialization/lexer
+
+import ../../waku/waku_api/rest/serdes
+
+type ProtocolTesterMessage* = object
```

Comment:
Interesting!
What is this type for?

Reply:
This is the payload of the messages light push client sends. It fills out on its own side to help calculating losses, mis-ordered receive, latency.
Also we will have multiple sender, to separate messages on the receiver side.